core/codesig/src/mill/codesig/ExternalSummary.scala

```diff
@@ -47,7 +48,8 @@ object ExternalSummary {
 
     def load(cls: JCls): Unit = methodsPerCls.getOrElse(cls, load0(cls))
 
-    def load0(cls: JCls): Unit = {
+    // Some macros implementations will fail the ClassReader, we can skip them
```

Comment:
What kind of error do those macro implementations produce? We should try and be specific about what errors we catch here, to avoid silencing unexpected errors that may indicate real issues